packages/arb-ts/src/lib/bridge.ts

```diff
@@ -567,4 +645,45 @@ export class Bridge {
       this.l1Bridge.l1Provider
     )
   }
+
+  public async setGateways(
+    tokenAddresses: string[],
+    gatewayAddresses: string[]
+  ) {
+    const gasPriceBid = await this.l2Bridge.l2Provider.getGasPrice()
+    const maxSubmissionPrice = (
+      await this.l2Bridge.getTxnSubmissionPrice(
+        300 + 20 * (tokenAddresses.length + gatewayAddresses.length)
```

Comment:
what does `300 + 20 *` mean?

Reply:
20 for each address size, 300 additional buffer for insurance

Reply:
should we add that as a comment in the code? And I'm not sure we need the 300 buffer insurance